app/models/condition.rb

```diff
@@ -189,8 +192,9 @@ def self._subst_find(rec, inputs, expr)
 
     if checkmode == "count"
       e = check.gsub(/<count>/i, list.length.to_s)
+      left, operator, right = e.split
       MiqPolicy.logger.debug("MIQ(condition-_subst_find): Check Expression after substitution: [#{e}]")
-      result = !!proc { $SAFE = 3; eval(e) }.call
+      result = !!left.to_f.send(operator, right.to_f)
```

Comment:
If `e` is guaranteed to be of the form `[float literal] [operator] [float literal]` at this point, can we just avoid constructing that string in the first place?

Where/what constraints do we apply to `operator` by this point? I don't imagine there's much you could call on a Float, while passing a float, that would be dangerous... but an apparently-unconstrained `send` is still eye-catching.